### PR TITLE
HTML-CSS-Popup — #done

### PR DESCRIPTION
**Pull request by Oleksandr-Heleta**
### Popup task

[Demo](https://oleksandr-heleta.github.io/kottans-popup/) |
[Code base](https://github.com/Oleksandr-Heleta/kottans-popup.git)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

File: submissions/Oleksandr-Heleta/popup/index.html [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+<body>
+    <header>
+        <nav>
+            <ul class="nav">
+                <li class="nav_item"><a href="#" target="_blank" rel="noopener noreferrer">Mail</a></li>
+                <li class="nav_item"><a href="#" target="_blank" rel="noopener noreferrer">Image</a></li>
+                <li class="nav_item">
+                    <a href="#" target="_blank">
+                        <img class="nav-img" src="./popup-icons/bell.png" alt="bell">
```
Comment (A-Ostrovnyy) [inline submissions/Oleksandr-Heleta/popup/index.html:21]: Read how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

File: submissions/Oleksandr-Heleta/popup/index.html [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+                    </a>
+                </li>
+                <li class="popup_button nav_item">
+
+                    <input class="popup_button-input" type="checkbox" name="menu" id="menu">
+                    <label class="popup_button-label" for="menu">
```
Comment (A-Ostrovnyy) [inline submissions/Oleksandr-Heleta/popup/index.html:27]: Add `cursor: pointer` to all interactive elements. Also, fix styles on hover and focus: avoid image shifting

File: submissions/Oleksandr-Heleta/popup/index.html [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+                        <img class="popup_button-img" src="./popup-icons/popup-button.png" alt="popup button">
+                    </label>
+
+
+                    <div class="popup">
+
+                        <input class="popup_more-input" type="checkbox" name="more" id="more_less">
```
Comment (A-Ostrovnyy) [inline submissions/Oleksandr-Heleta/popup/index.html:34]: When the popup is open, the first interactive element inside the popup must be in focus; in our case, when the user opens the popup and presses the Tab key, the first link should be in focus.

File: submissions/Oleksandr-Heleta/popup/index.html [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+                                    <p>Failes</p>
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/contacts.png" alt="contacts">
+                                    <p>Contacts</p>
+                                </a>
+                            </li>
+                            <li class="popup_item">
```
Comment (A-Ostrovnyy) [inline submissions/Oleksandr-Heleta/popup/index.html:77]: Fix icons alignment: just center it
image

File: submissions/Oleksandr-Heleta/popup/index.html [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+                                    <img class="popup_img" src="./popup-icons/apps.png" alt="apps">
+                                    <p>Apps</p>
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/tanslation.png" alt="tanslation">
+                                    <p>Translation</p>
+                                </a>
+                            </li>
```
Comment (A-Ostrovnyy) [inline submissions/Oleksandr-Heleta/popup/index.html:94]: Check and fix styles on focus: add straight borders
image

File: submissions/Oleksandr-Heleta/popup/index.html [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+                    <a href="#" target="_blank" rel="noopener noreferrer">
+                        <img class="nav-img" src="./popup-icons/favicon.png" alt="user">
+                    </a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+</body>
+
+
```
Comment (A-Ostrovnyy) [inline submissions/Oleksandr-Heleta/popup/index.html:165]: Remove unused lines

File: submissions/Oleksandr-Heleta/popup/index.html [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+
+
+
+
+</html>
```
Comment (A-Ostrovnyy) [inline submissions/Oleksandr-Heleta/popup/index.html:170]: Add an empty line at the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)

File: submissions/Oleksandr-Heleta/popup/style.css [hunk 8]
```diff
@@ -0,0 +1,213 @@
...
+@media (max-width:400px) {
+    .popup {
+        position: absolute;
+        right: -50px;
+        max-width: 300px;
+        min-width: 250px;
+
+    }
+
+}
```
Comment (A-Ostrovnyy) [inline submissions/Oleksandr-Heleta/popup/style.css:213]: Add an empty line at the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!

**Comment by Oleksandr-Heleta** [comment]
Please, review)

File: submissions/Oleksandr-Heleta/popup/index.html (hunk 4 above)
Comment (A-Ostrovnyy) [reply submissions/Oleksandr-Heleta/popup/index.html:71]: Still not fixed :cry:

File: submissions/Oleksandr-Heleta/popup/style.css [hunk 9]
```diff
@@ -0,0 +1,226 @@
...
+
+
+.popup_more-input,
+.popup_button-input {
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+
```
Comment (A-Ostrovnyy) [inline submissions/Oleksandr-Heleta/popup/style.css:25]: No need to add an empty line before closing brackets)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good progress!

File: submissions/Oleksandr-Heleta/popup/style.css (hunk 9 above)
Comment (Oleksandr-Heleta) [reply submissions/Oleksandr-Heleta/popup/style.css]: Have already deleted

File: submissions/Oleksandr-Heleta/popup/index.html [hunk 10]
```diff
@@ -0,0 +1,168 @@
...
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/videos.png" alt="">
+                                    <p>Video</p>
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/search.png" alt="">
```
Comment (A-Ostrovnyy) [inline submissions/Oleksandr-Heleta/popup/index.html:103]: Please fix the vertical alignment of the icons 
image

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
One last issue)

File: submissions/Oleksandr-Heleta/popup/index.html (hunk 10 above)
Comment (Oleksandr-Heleta) [reply submissions/Oleksandr-Heleta/popup/index.html:103]: I used flex-end. I hope it will be better). Please review one more

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**